edi_product_import_ubl/data/edi_exchange_type.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
As you decided to keep it, can you please put this in the roadmap? "Evaluate dropping this module in future versions since it's just a default configuration that can be described in docs."
TBH, the more I think / write about this, the more I'm not convinced we want to carry on this module :sweat_smile:

Reply:
@florentx ping :)